README.md

```diff
@@ -6,18 +6,6 @@ A serverless form submission application built and deployed to AWS with the Serv
 
 ![Architecture Diagram](./.images/architecture.png?raw=true)
 
-## Local Dev
```

Comment:
I think you've inadvertently removed a bunch of documentation.

Reply:
hmmm. from the readme?

Reply:
let me fix that

Reply:
done

Reply:
@joeaero19 When I look at the diff of the README in the Files Changed section, there's still quite a bit of stuff being removed.
I recommend you make a new branch (or reset this one) off of master, and add the handful of changes you need to.

Reply:
ok

Reply:
I compared with master readme file and added the diff.